checks/shell_download_validate_test.go

```diff
@@ -86,3 +88,18 @@ func TestIsSupportedShellScriptFile(t *testing.T) {
 		})
 	}
 }
+func TestValidateShellFile(t *testing.T) {
```

Comment:
If you just add a newline before this method and add
```
	t.Parallel()
```
at the beginning of this method, the linter should pass.

Other than that, looks great! Thanks for fixing this!

Reply:
My bad. I fixed it. Thanks.